src/Gui/CommandDoc.cpp

```diff
@@ -1154,13 +1154,13 @@ void StdCmdDelete::activated(int iMsg)
                             bodyMessageStream << "\n";
                         }
                         std::string thisDoc = pGuiDoc->getDocument()->getName();
-                        bodyMessageStream << qApp->translate("Std_Delete", 
-                                             "These items are selected for deletion, but are not in the active document. \n\n"); 
+                        bodyMessageStream << qApp->translate("Std_Delete",
+                                             "These items are selected for deletion, but are not in the active document.") << '\n\n';
                         for (const auto &currentLabel : inactiveLabels)
                           bodyMessageStream << currentLabel << " / " << Base::Tools::fromStdString(thisDoc) << '\n';
                     }
-                    bodyMessageStream << qApp->translate("Std_Delete",
-                                                         "\n\nAre you sure you want to continue?");
+                    bodyMessageStream << '\n\n' << qApp->translate("Std_Delete",
+                                                         "Are you sure you want to continue?");

```

Comment:
Not sure I did this correctly. I'm trying to eliminate superfluous whitespace from strings that are translated to avoid translators making mistakes.

Reply:
bf0be18